os/arch/arm/src/amebasmart/Kconfig

```diff
@@ -75,6 +75,25 @@ config AMEBASMART_USBDEVICE
 	bool "AMEBASMART_USBDEVICE"
 	default n
 
+config AMEBASMART_BOR
+	bool "AMEBASMART_BOR"
+	default y
+
+if AMEBASMART_BOR
+config AMEBASMART_Threshold_Fall
```

Comment:
Let's use upper characters all in the config name.

Reply:
Changed to all upper case, thank you.